ipn/conf.go

```diff
@@ -50,6 +50,9 @@ type ConfigVAlpha struct {
 	// should advertise amongst its wireguard endpoints.
 	StaticEndpoints []netip.AddrPort `json:",omitempty"`
 
+	RelayServerPort            *int             `json:",omitzero"`  // --relay-server-port
```

Comment:
with omitzero, I don't think we need this pointer anymore. (we're actively de-pointering such things elsewhere)

But if 0 vs unset has a distinction, that needs to be documented.

Also, I'd prefer these be actually documented, rather than just referring to some flag the reader is supposed to know.